scripts/apt

```diff
@@ -86,25 +67,15 @@ case "$1" in
 
 		for newrepo in "${ARGV[@]:1}"
 		do
-			echo "Removing preference for '$newrepo'"
-			sudo rm -f "/etc/apt/preferences.d/pop-os-staging-${newrepo//./_}"
-
-			echo "Removing repository for '$newrepo'"
-			LINE="deb ${REPOS}$newrepo ${UBUNTU_CODENAME} main"
-			if command -v apt-manage > /dev/null
-			then
-				sudo apt-manage remove -y "popdev-${LOCALREMOTE}${newrepo//./-}${DEVLINE}"
-			else
-				sudo add-apt-repository --enable-source --no-update --remove "${LINE}"
-			fi
+			echo "Deprecated: use 'apt-manage remove popdev:$newrepo' instead."
+			sudo apt-manage remove popdev:$newrepo
```

Comment:
Unless https://github.com/pop-os/repolib/issues/58 leads to a change, this probably won't work. Firstly because there will be a `-` instead of the `:`, and secondly because branch names get parsed by apt-manage, so `linux-6.0.2` would be converted into `popdev-linux-6-0-2`, for example.

Reply:
I tested it and it does work. Regarding https://github.com/pop-os/repolib/issues/58, even now (as of https://github.com/pop-os/repolib/pull/54) using a colon has apt-manage autocorrect/suggest a hyphen instead (which is a pretty clear indicator of how confusing the current setup is, imo.) Using a colon just means you have to press Enter an extra time to accept the autocorrect.

> secondly because branch names get parsed by apt-manage, so `linux-6.0.2` would be converted into `popdev-linux-6-0-2`, for example.

I don't understand what your concern is at all here. What you're describing is the correct behavior. The old syntax would have been:

```
./pop/scripts/apt add linux-6.0.2
./pop/scripts/apt remove linux-6.0.2
```

And the new syntax (with the current inconsistent syntax) is:

```
apt-manage add popdev:linux-6.0.2
apt-manage remove popdev-linux-6.0.2
```

Reply:
>And the new syntax (with the current inconsistent syntax) is:
>```
>apt-manage add popdev:linux-6.0.2
>apt-manage remove popdev-linux-6.0.2
>```

New syntax would be:
```
apt-manage add popdev:linux-6.0.2
apt-manage remove popdev-linux-6-0-2
```

It's another situation where the correction works, but I'm just pointing out two instances of the same point I was making. Like I said in my comment below, this is probably acceptable, but I don't want to be the only one making that decision.

Reply:
> It's another situation where the correction works,

If autocorrection works, then it's working. (I did just test this with `linux-6.0.2`, and it does work.) This is definitely an improvement over the current completely broken behavior.

The questionable `apt-manage` semantics requiring autocorrect for predictable syntax aside, the alternative here would be to have the `apt` script strip out periods and replace them with dashes itself. I would argue it's safer to leave all substitution to `apt-manage` since it knows what format it's expecting.

Reply:
Sorry, I think saying "won't work" conveyed the wrong idea. I'm saying that we're feeding something to apt-manage that it will complain about. If it didn't have such helpful spelling recommendations, then it would be non-working. 

My main point was that since it's relying on those spelling fixes and displays a deprecation warning, this is probably still an okay solution. I just wanted it on peoples' radar since it was something I noticed.

If there are no objections in the next half hour or so, I will approve and merge it.